W3ChampionsIdentificationService/Middleware/HttpException.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Net;
+
+namespace W3ChampionsIdentificationService.Middleware
+{
+    public class HttpException : Exception
+    {
+        private readonly int httpStatusCode;

```

Comment:
naming convention is to have the readonly field with _ => `private readonly int _httpStatusCode`